Board.js

```diff
@@ -17,8 +21,9 @@ class Board {
 	* @param table The DOM element to render the board to
 	* @param showShips Boolean for whether all ship locations should be visible
 	* @param executive Object to use the clickSpace method of
+	* @param final Boolean for whether the game is already won
 	**/
-	render(table, showShips, executive) {
+	render(table, showShips, executive, final) {
```

Comment:
Since showShips and final are both booleans describing how the board will render, they should be next to each other in the parameter list.